README.md

```diff
@@ -14,29 +14,6 @@ This repository includes files with plain SQL that can be used to recreate a dat
 
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/224459045-45951d12-4004-4129-bd48-35e01adc13ba.png)

- [ ] You are not following Gitflow which is a crucial requirement of your projects. Kindly use the correct Gitflow :+1: